src/public/Interfaces.ts

```diff
@@ -95,6 +95,53 @@ export interface HttpRequest {
      * The HTTP request body as a UTF-8 string.
      */
     rawbody?: any;
+    /**
+     * Claims-based security information for the current HTTP request.
```

Comment:
A little bit confused by the wording of this, would something like this make sense?
"Claims-based identities for authenticated HTTP requests."